brigade/plugins/tasks/connections/netmiko_connection.py

```diff
@@ -0,0 +1,46 @@
+from netmiko import ConnectHandler
+
+napalm_to_netmiko_map = {
+    'ios': 'cisco_ios',
+    'nxos': 'cisco_nxos',
+    'eos': 'arista_eos',
+    'junos': 'juniper_junos',
+    'iosxr': 'cisco_iosxr'
+}
+
+
+def netmiko_connection(task=None, host=None, **netmiko_args):
+    """Connect to the host using Netmiko and set the relevant connection in the connection map.
+
```

Comment:
Should we add a note about `netmiko_host` mapping to `host` in netmiko's ConnectHandler?